ts/features/newProfile/store/actions/optInActions.ts

```diff
@@ -0,0 +1,8 @@
+import { ActionType, createStandardAction } from "typesafe-actions";
+import { NewProfileOptInState } from "../reducers/optInReducer";
+
+export const setNewProfileOptIn = createStandardAction(
```

Comment:
What is the scope of this action?

Reply:
Is a opt-in mandatory?

Reply:
The scope of the action should be to opt in for the new Profile option. It's mandatory in the sense of being able of see the new profile, it should be mandatory to switch the toggle on, if I understood correctly